.github/workflows/cicd-pipeline.yml

```
@@ -60,23 +62,25 @@ jobs:
       - name: Build bootJar
         run: ./gradlew :clody-app:bootJar --no-daemon
 
-      - name: Login to DockerHub
-        run: echo "$DOCKERHUB_PASSWORD" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+      # AWS 인증
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)